### ISSUE TITLE
train code??

### ISSUE THREAD

**Issue by MingiJi** [luckybird1994/classnet#2]
Could you share code for exact loss function in the paper??
Is there any hyperparameters for object, region and pixel loss?

**Comment by CVPR2021Submission2094** [comment]
> Could you share code for exact loss function in the paper??
> Is there any hyperparameters for object, region and pixel loss?

I have already shared the code of loss fuctions in this project。You can find in "pytorch_wasserstein" folder and "train.py" file. The hyperparameters of these three loss functions are proposed in paper.

**Comment by luckybird1994** [comment]
I have already shared the code of loss fuctions in this project。You can find in "pytorch_wasserstein" folder and "train.py" file. The hyperparameters of these three loss functions are proposed in paper.
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
